src/state/index.ts

```diff
@@ -19,4 +19,27 @@ export const reducers = (history: History<any>): { [key in keyof AppState]: Redu
 
 export const history = createBrowserHistory({ basename: 'localsurvey' });
 
-export const store = createStore(combineReducers(reducers(history)), composeWithDevTools());
+// Define a preloadedState for createStore()
+// See: https://redux.js.org/recipes/structuring-reducers/initializing-state
+const preloadedState: any = {
```

Comment:
I prefer having this strongly typed & in a separate file, then we can load it in conditionally here without bloating the file

Reply:
You can type it as `Partial<AppState>`

Reply:
Will update. Thank you.